includes/config.defaults.php

```diff
@@ -69,7 +69,9 @@ function set_defaults_for_routers(&$parsed_config) {
     // Show visitor IP address
     'show_visitor_ip' => true,
     // Frontpage order you can use: routers, commands, parameter, buttons
-    'order' => array('routers', 'commands', 'parameter', 'buttons')
+    'order' => array('routers', 'commands', 'parameter', 'buttons'),
+    // Number of routers to show on frontpage
+    'router_count' => 5
```

Comment:
if we use this:
`elseif  ($this->frontpage['router_count'] > 0 AND $this->frontpage['router_count'] <= 5 }`
we dont need for:
`'router_count' => 5`